shell/server/drivers/ip.js

```diff
@@ -121,9 +122,14 @@ IpInterfaceImpl = class IpInterfaceImpl {
   }
 };
 
-makeIpInterface = (persistentMethods) => {
-  return new Capnp.Capability(new IpInterfaceImpl(persistentMethods), IpRpc.PersistentIpInterface);
-};
+// TODO(cleanup): Meteor.startup() needed because 00-startup.js runs *after* code in subdirectories
+//   (ugh).
```

Comment:
Maybe, but also I intend to move these pseudo-drivers to imports, at which point this particular ugliness will go away.